Upstream: No, disables parts of test suite
Source: William Orr <will@worrbase.com>
Reason: test/simple/test-net-server-bind specifically tests not passing a host, which defaults to 0.0.0.0

--- node-orig/test/simple/test-net-server-bind.js	2012-08-22 14:03:20.000000000 -0400
+++ node-v0.8.8/test/simple/test-net-server-bind.js	2012-08-29 18:05:26.502702698 -0400
@@ -41,7 +41,7 @@
 var address1;
 var server1 = net.createServer(function(socket) { });
 
-server1.listen(common.PORT);
+server1.listen(common.PORT, "127.0.0.1");
 
 setTimeout(function() {
   address1 = server1.address();
@@ -55,7 +55,7 @@
 var address2;
 var server2 = net.createServer(function(socket) { });
 
-server2.listen(common.PORT + 1, function() {
+server2.listen(common.PORT + 1, "127.0.0.1", function() {
   address2 = server2.address();
   console.log('address2 %j', address2);
   server2.close();
@@ -67,29 +67,16 @@
 var address3;
 var server3 = net.createServer(function(socket) { });
 
-server3.listen(common.PORT + 2, '0.0.0.0', 127, function() {
+server3.listen(common.PORT + 2, '127.0.0.1', 127, function() {
   address3 = server3.address();
   console.log('address3 %j', address3);
   server3.close();
 });
 
 
-// Backlog argument without host argument
-
-var address4;
-var server4 = net.createServer(function(socket) { });
-
-server4.listen(common.PORT + 3, 127, function() {
-  address4 = server4.address();
-  console.log('address4 %j', address4);
-  server4.close();
-});
-
-
 process.on('exit', function() {
   assert.ok(address0.port > 100);
   assert.equal(common.PORT, address1.port);
   assert.equal(common.PORT + 1, address2.port);
   assert.equal(common.PORT + 2, address3.port);
-  assert.equal(common.PORT + 3, address4.port);
 });
